dataplicity/client.py

```diff
@@ -153,74 +152,79 @@ def _sync(self):
                     batch.call_with_id(
-                        'authenticate_result',
-                        'device.check_auth',
-                        device_class='tuxtunnel',
+                        "authenticate_result",
+                        "device.check_auth",
+                        device_class="tuxtunnel",
                         serial=self.serial,
                         auth_token=self.auth_token,
-                        sync_id=sync_id
+                        sync_id=sync_id,
                     )
                     self._sync_meta(batch)
-                batch.get_result('authenticate_result')
+                batch.get_result("authenticate_result")
                 self._check_meta(batch)
 
         finally:
             elapsed = time.time() - start
-            log.debug('sync complete %0.2fs', elapsed)
+            log.debug("sync complete %0.2fs", elapsed)
 
     def _sync_meta(self, batch):
         """Sync meta information regarding host device."""
         try:
             meta = device_meta.get_meta()
             log.debug("syncing meta %r", meta)
         except:
-            log.exception('error getting meta')
+            log.exception("error getting meta")
         else:
             batch.call_with_id(
-                'set_agent_version_result',
-                'device.set_agent_version',
-                agent_version=meta['agent_version']
+                "set_agent_version_result",
+                "device.set_agent_version",
+                agent_version=meta["agent_version"],
             )
             batch.call_with_id(
-                'set_machine_revision_result',
-                'device.set_machine_revision',
-                revision_code=meta['machine_revision']
+                "set_machine_revision_result",
+                "device.set_machine_revision",
+                revision_code=meta["machine_revision"],
             )
             batch.call_with_id(
-                'set_os_version_result',
-                'device.set_os_version',
-                os_version=meta['os_version']
+                "set_os_version_result",
+                "device.set_os_version",
+                os_version=meta["os_version"],
             )
             batch.call_with_id(
-                'set_uname_result',
-                'device.set_uname',
-                uname=meta['uname']
+                "set_uname_result", "device.set_uname", uname=meta["uname"]
             )
             batch.call_with_id(
-                'set_ip_addresses',
-                'device.set_ip_addresses',
-                ip_list=meta['ip_list']
+                "set_ip_addresses_result",
+                "device.set_ip_addresses",
+                ip_list=meta["ip_list"],
+            )
+            batch.call_with_id(
+                "set_machine_defined_tags_result",
+                "device.set_machine_defined_tags",
+                tag_list=meta["tag_list"],
```

Comment:
Black has converted all the single quotes to double quotes.

Here is the crux of the actual work here.